torch/csrc/jit/passes/quantization.cpp

```diff
@@ -1,6 +1,7 @@
 #include <torch/csrc/jit/passes/quantization.h>
 #include <torch/csrc/jit/passes/subgraph_rewrite.h>
 #include <torch/csrc/jit/passes/constant_propagation.h>
+#include <torch/csrc/jit/passes/fuse_linear.h>
```

Comment:
I suspect clang-format would like to rearrange the headers in the alphabetical order - please run clang-format on all changes (the constant_propagation header added in the previous PR should be reformatted in the corresponding PR too).